ezbake/config/conf.d/webserver.conf

```diff
@@ -1,5 +1,6 @@
 webserver: {
+    access-log-config = /etc/puppetserver/request-logging.xml
```

Comment:
Jeremy's suggestion is correct

(https://github.com/puppetlabs/puppet-specifications/blob/master/file_paths.md#puppetserver)